custom_components/garmin_connect/services.py

```
@@ -0,0 +1,157 @@
+ADD_BLOOD_PRESSURE_SCHEMA = vol.Schema(
+    {
+        vol.Required("systolic"): int,
+        vol.Required("diastolic"): int,
+        vol.Required("pulse"): int,
+        vol.Optional("timestamp"): cv.string,
+        vol.Optional("notes"): cv.string,
+    }
+)
+
+
+def _get_coordinator(hass: HomeAssistant):
```

Ruff (0.14.10)

Missing return type annotation for private function `_get_coordinator`

(ANN202)